src/component/Board.jsx

```diff
@@ -51,7 +68,8 @@ function Board() {
   }
   // function to check game is ended or not
   const gameEnd = () => {
-    if (pairCounter >= cardLevel / 2) {
+    if (pairCounter >= cardLevel / 2 - 1) {
+      setHeading(true);
```

Comment:
Having `heading` as `boolean` is tricky, I would suggest to have it as direct string like

initially heading will be "Memory Game"
Later when game over you can change it to "Congratulation you Won!"

Reply:
Implemented!